### PR TITLE
Add example manifests for queue/exchange/binding

### PR DESCRIPTION
**Pull request by ChunyiLyu**
**Note to reviewers:** remember to look at the commits in this PR and consider if they can be squashed

## Summary Of Changes

- examples for queue/exchange/binding

### CODE REVIEW

File: docs/examples/bindings/binding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rabbitmq.com/v1alpha1
+kind: Binding
+metadata:
+  name: binding
+spec:
+  vhost: "/test-vhost" # default to '/' if not provided
+  source: test # an exisitng exchange
```
Comment (coro) [inline docs/examples/bindings/binding.yaml:7]: typo: existing

File: docs/examples/queues/quorum-queue.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+kind: Queue
+metadata:
+  name: qq-example
+spec:
+  name: test # name of the queue
+  vhost: "/test-vhost" # default to '/' if not provided
+  rabbitmqClusterReference:
+    name: test
+    namespace: rabbitmq-system
+  type: quorum # withouth providing a queue type, rabbitmq creates a classic queue
```
Comment (coro) [inline docs/examples/queues/quorum-queue.yaml:11]: typo: 'without'

**Review by coro — APPROVED** [review]
LGTM, just a couple typos. I'll add the user one in my PR

File: docs/examples/bindings/binding.yaml (hunk 1 above)
Comment (ChunyiLyu) [reply docs/examples/bindings/binding.yaml:7]: 😆  I should publish a dictionary

**Pull request merged**